content/blog/2024-12-21-how-do-i-...-in-dt-5.0/index.md

```diff
@@ -0,0 +1,508 @@
+any processing module in darktable can be [masked](https://darktable.org/usermanual/en/darkroom/masking-and-blending/).
+
+</details>
+
+<details><summary><b>apply a preset</b></summary>
+
+What other programs call presets, darktable calls _styles_.
+
+In darktable, a _preset_ is a particular configuration of a __single__
+module, which may be accessed by clicking on the hamburger icon at the
+right of the module's header.
+
+A _style_ is a collection of presets for one or more modules which can
+be applied to one or more images at once.  You can apply styles in
+lighttable view with the
+[_styles_](https://darktable.org/usermanual/en/module-reference/utility-modules/lighttable/styles/)
+module, or in darkroom view by clicking on the styles quick access
+button ![styles](styles-icon.png#icon) just below the bottom left corner of the image.
+
+</details>
+
+<details><summary><b>adjust hue/saturation/lightness (HSL)</b></summary>
+
+To adjust the hue, saturation, and/or lightness of pixels based on
+their original colors, use the
+[_color equalizer_](https://darktable.org/usermanual/en/module-reference/processing-modules/color-equalizer/)
+module.
+
+To adjust the hue, saturation, and/or lightness of pixels based on
+their saturation or lightness, use the
+[_color zones_](https://darktable.org/usermanual/en/module-reference/processing-modules/color-zones/)
+module.
+
+</details>
+
+<details><summary><b>crop/straighten an image</b></summary>
+
+To crop an image, open the
+[_crop_](https://darktable.org/usermanual/en/module-reference/processing-modules/crop/)
+module and drag the edges/corners of the crop box drawn on your image.
+
+To straighten an image, right-click on your image and drag along a
+line which _should_ be horizontal or vertical, then release the mouse
+button.  darktable will then rotate your image to make the line along
+which you dragged exactly horizontal or vertical.  You can also use
+the _rotation_ slider in the
+[_rotate and perspective_](https://darktable.org/usermanual/en/module-reference/processing-modules/rotate-perspective/),
+which is shown in the
+[_Quick Access Panel_](https://darktable.org/usermanual/en/darkroom/organization/quick-access-panel/)
+by default.
+
+To rotate the image by multiples of 90 degrees, use the
+[_orientation_](https://darktable.org/usermanual/en/module-reference/processing-modules/orientation/)
+module.
+
+</details>
+
+<details><summary><b>adjust contrast/brightness/saturation</b></summary>
+
+To adjust contrast, you can use
+
+- the contrast slider in either
+  [_filmic rgb_](https://darktable.org/usermanual/en/module-reference/processing-modules/filmic-rgb/)'s
+  _look_ tab or [_sigmoid_](https://darktable.org/usermanual/en/module-reference/processing-modules/sigmoid/)
+  (on the [_Quick Access Panel_](https://darktable.org/usermanual/en/darkroom/organization/quick-access-panel/)
+  by default when workflow is set to either of the "scene-referred" options in preferences).
+
```

Comment:
why no tone equalizer here? Its way more powerful than whats in color balance rgb?

Reply:
It's also a lot more complicated than moving a single slider as can be done with the options listed.